keps/sig-node/4569-cgroup-v1-deprecation/README.md

```diff
@@ -0,0 +1,651 @@
+# KEP-4569: Remove Support for cgroup v1 in Kubernetes
```

Comment:
Whoa there. Are we removing or deprecating? The PR title (“Deprecate cgroup v1 support”) is not proposing the same thing as this text, and it's a **huge** difference.

Deprecating is relatively easy (and we probably should do that). The hard thing to manage is the removal.

The title of issue 4569 is also “Deprecate support for cgroup v1”. It's such an important difference to highlight that I feel confident to apply a hold to call it out.

/hold
pending resolution of whether this KEP is calling for deprecation only, or for ultimate removal with deprecation as a step on that journey.

Reply:
(Wikipedia's explanation of [deprecation](https://en.wikipedia.org/wiki/Deprecation#Software))

Reply:
Thanks for the wikipedia definition, we started off with the idea of deprecation only but changed the wording a bit later. 

https://github.com/kubernetes/enhancements/pull/4572#discussion_r1554395982

cc @SergeyKanzhelev

Reply:
```suggestion
# KEP-4569: Moving cgroup v1 support into maintenance mode
```

(BTW “Move” would be more idiomatic, but doesn't match the KEP issue title)